test/TEST-04-JOURNAL/test-journal.sh

```diff
@@ -51,6 +51,14 @@ journalctl --sync
 journalctl -b -o cat -t "$ID" >/output
 cmp /expected /output
 
+# --output-fields restricts output
+ID=$(journalctl --new-id128 | sed -n 2p)
+printf $'foo' | systemd-cat -t "$ID" --level-prefix false
+journalctl --sync
+journalctl -b -o export --output-fields=MESSAGE,FOO --output-fields=PRIORITY,MESSAGE -t "$ID" >/output
+# cannot compare all fields (contain times and cursor), but ensure number of lines
+[[ `grep -c . /output` -eq 6 ]]
```

Comment:
Could this also assert via a `grep -q` that the MESSAGE and PRIORITY fields are present, but e. g. FOO= and SYSLOG_FACILITY= aren't? This would more directly ensure that this actually works as intended. Thanks!

Reply:
You're right of course. I've added those checks in addition to counting the lines. Thanks!